src/index.js

```diff
@@ -33,17 +36,39 @@ PrivateRoute.propTypes = {
   location: PropTypes.string
 };
 
+const TopBar = () => (
+  <div className="topBar">
+    <Link className="gameLink" to="/game">
+      Game
+    </Link>
+    <Link className="profileLink" to="/profile">
+      Profile
+    </Link>
+    <Link className="signoutLink" to="/signout">
+      SignOut
+    </Link>
+  </div>
+);
+
+const Empty = () => <div>Empty Page</div>;
+
 ReactDOM.render(
   <Provider store={store}>
-    <Fragment>
-      <Router history={history}>
-        <Switch>
-          <Route exact path="/" render={() => <Redirect to="/login" />} />
-          <Route path="/login" component={LogIn} />
-          <PrivateRoute path="/game" component={Game} />
-        </Switch>
-      </Router>
-    </Fragment>
+    <Router history={history}>
+      <Switch>
+        <Route exact path="/" render={() => <Redirect to="/login" />} />
+        <Route path="/login" component={LogIn} />
+        <PrivateRoute path="/game" component={Game} />
+        <PrivateRoute path="/profile" component={Empty} />
+        <Route
```

Comment:
dont need a route to do the logout, just use the sign out button and dispatch an action that remove token and push login view

Reply:
Done